frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideInBelowAction.java

```diff
@@ -126,9 +126,9 @@ public void actionPerformed(AnActionEvent anActionEvent) {
     }
 
     // TODO (#430): expose getParent from branch layout API
```

Comment:
dont we want to close it btw? can we still call it "branch layout API" anyway?

Reply:
#430 is done on #1264, to facilitate #1243

Reply:
Let's stick to `branch layout API` naming 🤔